package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "generator-ibm-web",
-  "version": "1.2.6",
+  "name": "generator-web-golang",
```

Comment:
revert this change back to generator-ibm-web

Reply:
also make sure version is reverted. You don't need to increment the version, an auto-PR will be created to update it.